strr-base-web/app/interfaces/strr-api.ts

```diff
@@ -134,6 +134,7 @@ export interface ApiDocument {
   fileType: string
   uploadStep?: DocumentUploadStep
   uploadDate?: string
+  addedOn?: string
```

Comment:
Name is a bit unclear... what this `addedOn` represents?

Reply:
What about the `uploadDate`? Isn't it the same?

Reply:
Done